Story/StoryLobbyData.cs

```diff
@@ -37,6 +38,13 @@ public class State : ResourceDataState
             public int quarterfood;
             [OnlineField]
             public int mushroomCounter;
+            [OnlineField]
```

Comment:
Ok, I am requiring all players to have Remix enabled to play online for three reasons:

1. It's free
2. Enabling it does not alter the user experience much, and is configurable to be completely turned off while enabled.
3. It's very few line of code change versus writing more logic for host / client behaviors. 


Provided you are on board with the argument, I will proceed with changing the logic to become an RPC since it's ephemeral.